deepinv/loss/scheduler.py

```diff
@@ -89,8 +89,9 @@ class RandomLossScheduler(BaseLossScheduler):
     :Example:
 
     >>> import torch
-    >>> from deepinv.loss import RandomLossScheduler, SupLoss, SSIM
-    >>> l = RandomLossScheduler(SupLoss(), SSIM()) # Choose randomly between Sup and SSIM
+    >>> from deepinv.loss import RandomLossScheduler, SupLoss
+    >>> from deepinv.loss.metric import SSIM
+    >>> l = RandomLossScheduler(SupLoss(), SSIM(train_loss=True)) # Choose randomly between Sup and SSIM
```

Comment:
see issue with train_loss option above - I'm okay with having a special kwarg for SSIM